mantle/kola/tests/rpmostree/deployments.go

```diff
@@ -30,13 +30,15 @@ func init() {
 		Run:         rpmOstreeUpgradeRollback,
 		ClusterSize: 1,
 		Name:        "rpmostree.upgrade-rollback",
+		Description: "Verify an upgrade and rollback with a simulated update works.",
 		FailFast:    true,
 		Tags:        []string{"rpm-ostree", "upgrade"},
 	})
 	register.RegisterTest(&register.Test{
 		Run:         rpmOstreeInstallUninstall,
 		ClusterSize: 1,
 		Name:        "rpmostree.install-uninstall",
+		Description: "Verifies rpm-ostree supports installing and uninstalling rpms.",
```

Comment:
Minor: Verify (for consistency).

We can fix it in a follow-up. It's not worth the respin.